book/configuration.md

```diff
@@ -59,6 +86,33 @@ prediction = model.predict(variables_test, constant_a=prediction_parameters["con
 prediction.to_csv(output_path)
 ```
 
+```{code-tab} r R
+# Configuration
```

Comment:
again, add a note saying it's not good practice

Reply:
Fixed.